Makefile

```diff
@@ -195,7 +191,7 @@ version_file:
 	if [ "$(VENV_BASE)" ]; then \
 		. $(VENV_BASE)/awx/bin/activate; \
 	fi; \
-	python -c "import awx as awx; print awx.__version__" > /var/lib/awx/.awx_version; \
+	python -c "import awx as awx; print(awx.__version__)" > /var/lib/awx/.awx_version; \
```

Comment:
Isn't `import awx as awx` redundant? Maybe it could be just `import awx`, doesn't it?

Reply:
Ha, good eye :)

If I had to guess, this is probably a leftover from our `tower -> awx` renaming when we open sourced.